GooglePlayInstant/Editor/PlayInstantLoadingScreenGenerator.cs

```diff
@@ -32,19 +32,26 @@ public class PlayInstantLoadingScreenGenerator
         private const string LoadingScreenJsonFileName = "LoadingScreenConfig.json";
 
         private static readonly string LoadingScreenScenePath =
-            Path.Combine("Assets", "PlayInstantLoadingScreenAssets");
+            Path.Combine("Assets", "PlayInstantLoadingScreen");
 
-        private static readonly string LoadingScreenJsonPath = Path.Combine(LoadingScreenScenePath, "Resources");
+        private static readonly string LoadingScreenResourcesPath = Path.Combine(LoadingScreenScenePath, "Resources");
 
-        public static string LoadingScreenImagePath;
+        private static string _loadingScreenImagePath;
 
         /// <summary>
         /// Sets the loadingScreenImagePath target for future generated loading screens.
         /// </summary>
-        public static void SetLoadingScreenImagePath()
+        public static void SetLoadingScreenImagePath(string loadingScreenImagePath)
         {
-            LoadingScreenImagePath =
-                EditorUtility.OpenFilePanel("Select Image", "", "png,jpg,jpeg,tif,tiff,gif,bmp");
+            _loadingScreenImagePath = loadingScreenImagePath;
+        }
+        
+        /// <summary>
+        /// Returns the loadingScreenImagePath string
+        /// </summary>
+        public static string GetLoadingScreenImagePath()
```

Comment:
This method and the one above can be replaced with a C# auto-implemented property:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties

public static string LoadingScreenImagePath { get; set; }

Reply:
How cool! Resolved.